app/js/dataModels/DiseaseModel.js

```diff
@@ -102,8 +102,7 @@ function factoryWrapper($log, _, $q, $timeout, $http, AppSettings){
   // simple math for the number of samples without a mutated gene 
   // matching the user selected genes
   DiseaseModel.prototype._setNegatives = function(){
-    
-     this.negatives = this.positives ?  (this.samples - this.positives) : null;
+     this.negatives = this.samples - this.positives;
```

Comment:
It seemed to me from looking at this: https://github.com/dcgoss/frontend/blob/e722e4fbddcd7f53fef29e9d6e797525ab1e6b45/app/js/dataModels/DiseaseModel.js#L16-L20 that everything is set to 0 to begin with